tidb-cloud/connected-care-detail.md

```diff
@@ -0,0 +1,157 @@
+
+This document provides detailed information about the [Connected Care](/tidb-cloud/connected-care-announcement.md) support services.
+
+<table>
+<thead>
+  <tr>
+    <th></th>
+    <th>Basic</th>
+    <th>Developer</th>
+    <th>Enterprise</th>
+    <th>Premium</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Features and Services</td>
+    <td>Great for personal starter projects, connecting you with an active community and AI-powered assistance for essential support needs.</td>
+    <td>Ideal for development and testing phases, offering direct connections and unlimited access to tech support, to kickstart your cloud journey.</td>
+    <td>Designed for production workloads, connecting your operations with advanced monitoring, prompt alerts, and fast 7x24 response times for a seamless experience.</td>
+    <td>Built for enterprises with mission-critical workloads, offering the fastest response times, proactive support, and dedicated connections through named Technical Account Managers.</td>
+  </tr>
+  <tr>
+    <td>Pricing</td>
+    <td>Free</td>
+    <td>Monthly cost is based on the greater of $29 or 10% of total monthly charges.</td>
+    <td>Monthly cost is based on the greater of $699 or 20% of total monthly charges.</td>
+    <td>Contact sales</td>
+  </tr>
+  <tr>
+    <td rowspan="4">Initial Response Time</td>
+    <td>-</td>
+    <td>S1: 4 business hours</td>
+    <td>S1: 1 hour</td>
+    <td>S1: 30 minutes</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>S2: 8 business hours</td>
+    <td>S2: 2 hours</td>
+    <td>S2: 1 hour</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>S3: 12 business hours</td>
+    <td>S3: 8 business hours</td>
+    <td>S3: 4 business hours</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>S4: 24 business hours</td>
+    <td>S4: 24 business hours</td>
+    <td>S4: 24 business hours</td>
+  </tr>
+  <tr>
+    <td rowspan="10">Services</td>
+    <td>Billing and account support</td>
+    <td>Billing and account support</td>
+    <td>Billing and account support</td>
+    <td>Billing and account support</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>Ticket-based technical support</td>
+    <td>Ticket-based technical support</td>
+    <td>Ticket-based technical support</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>End-to-end database guidance</td>
+    <td>End-to-end database guidance</td>
+    <td>End-to-end database guidance</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: AI chat in IM</td>
+    <td>Connected: AI chat in IM</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: IM subscription for TiDB Cloud alerts</td>
+    <td>Connected: IM subscription for TiDB Cloud alerts</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: IM subscription for support ticket updates</td>
+    <td>Connected: IM subscription for support ticket updates</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: IM ticket creation</td>
```

Comment:
Please see if we are able to adjust the column length for better visualization. 
Currently, it looks like one takes two rows and the other takes only one row
<img width="483" alt="Screenshot 2025-01-14 at 2 48 52 PM" src="https://github.com/user-attachments/assets/ce161226-d83f-4146-8d79-b35f87669486" />

Reply:
Let me create a preview of this PR and check it.

Reply:
In the [preview](https://pr.pingcap-docsite-preview.pages.dev/tidbcloud/connected-care-detail), the column length looks the same.

![image](https://github.com/user-attachments/assets/6aeeb3bf-53a8-494d-aaa6-623e6bb02d24)